### PR TITLE
Update to blocklib 0.1.8

### PR DESCRIPTION
**Pull request by wilko77**
new blocklib required some code changes.

### CODE REVIEW

File: anonlinkclient/utils.py [hunk 1]
```diff
@@ -85,42 +86,37 @@ def generate_candidate_blocks_from_csv(input_f: TextIO,
...
     block_state_vars = {}  # type: Dict[str, Any]
     state = blocking_obj.state
+
+    def base_model_to_dict(value):
+        if isinstance(value, list):
+            return [base_model_to_dict(item) for item in value]
+        if isinstance(value, BaseModel):
+            return value.dict()
+        else:
+            return value
```
Comment (hardbyte) [inline anonlinkclient/utils.py:114]: Is this because blocklib is now exposing pydantic BaseModels? That doesn't seem ideal...

**Review by hardbyte — APPROVED** [review]
Let's merge this and then come back to the CI

File: anonlinkclient/utils.py (hunk 1 above)
Comment (wilko77) [reply anonlinkclient/utils.py:114]: yes. I was wondering why we want to add the 'state' to the meta data in the first place? It seems to be just a repeat of the config, which has a separate section in the meta data.

Comment (hardbyte) [reply anonlinkclient/utils.py:114]: Yeah, blocklib needs a bit more love

**Pull request merged**